backend/src/main/java/com/cgi/retrospecto/backend/poker/controller/dto/StoryDTO.java

```diff
@@ -0,0 +1,37 @@
+package com.cgi.retrospecto.backend.poker.controller.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StoryDTO {
```

Comment:
What is the difference between StoryDTO and Story? The fields are the same, usually we use DTO-s to hide information from the frontend, not expose something that is not needed, but in this case these classes are same, so using the keep it simple principle this dto could be avoided.

Reply:
Removed StoryDTO.